src/physics/include/grins/physics_naming.h

```diff
@@ -60,140 +60,144 @@ namespace GRINS
-    { return "BoussinesqBuoyancyAdjointStabilization"+_suffix; }
+    { return "BoussinesqBuoyancyAdjointStabilization"+_suffix(); }
 
     static PhysicsName boussinesq_buoyancy_spgsm_stab()
-    { return "BoussinesqBuoyancySPGSMStabilization"+_suffix; }
+    { return "BoussinesqBuoyancySPGSMStabilization"+_suffix(); }
 
     static PhysicsName axisymmetric_boussinesq_buoyancy()
-    { return "AxisymmetricBoussinesqBuoyancy"+_suffix; }
+    { return "AxisymmetricBoussinesqBuoyancy"+_suffix(); }
 
     static PhysicsName low_mach_navier_stokes()
-    { return "LowMachNavierStokes"+_suffix; }
+    { return "LowMachNavierStokes"+_suffix(); }
 
     static PhysicsName low_mach_navier_stokes_braack_stab()
-    { return "LowMachNavierStokesBraackStabilization"+_suffix; }
+    { return "LowMachNavierStokesBraackStabilization"+_suffix(); }
 
     static PhysicsName low_mach_navier_stokes_spgsm_stab()
-    { return "LowMachNavierStokesSPGSMStabilization"+_suffix; }
+    { return "LowMachNavierStokesSPGSMStabilization"+_suffix(); }
 
     static PhysicsName low_mach_navier_stokes_vms_stab()
-    { return "LowMachNavierStokesVMSStabilization"+_suffix; }
+    { return "LowMachNavierStokesVMSStabilization"+_suffix(); }
 
     static PhysicsName reacting_low_mach_navier_stokes()
-    { return "ReactingLowMachNavierStokes"+_suffix; }
+    { return "ReactingLowMachNavierStokes"+_suffix(); }
 
     static PhysicsName elastic_membrane()
-    { return "ElasticMembrane"+_suffix; }
+    { return "ElasticMembrane"+_suffix(); }
 
     static PhysicsName elastic_cable()
-    { return "ElasticCable"+_suffix; }
+    { return "ElasticCable"+_suffix(); }
 
     static PhysicsName elastic_cable_rayleigh_damping()
-    { return "ElasticCableRayleighDamping"+_suffix; }
+    { return "ElasticCableRayleighDamping"+_suffix(); }
 
     static PhysicsName elastic_membrane_rayleigh_damping()
-    { return "ElasticMembraneRayleighDamping"+_suffix; }
+    { return "ElasticMembraneRayleighDamping"+_suffix(); }
 
     static PhysicsName elastic_membrane_constant_pressure()
-    { return "ElasticMembraneConstantPressure"+_suffix; }
+    { return "ElasticMembraneConstantPressure"+_suffix(); }
 
     static PhysicsName elastic_cable_constant_gravity()
-    { return "ElasticCableConstantGravity"+_suffix; }
+    { return "ElasticCableConstantGravity"+_suffix(); }
 
     static PhysicsName constant_source_term()
-    { return "ConstantSourceTerm"+_suffix; }
+    { return "ConstantSourceTerm"+_suffix(); }
 
     static PhysicsName parsed_source_term()
-    { return "ParsedSourceTerm"+_suffix; }
+    { return "ParsedSourceTerm"+_suffix(); }
 
     static PhysicsName convection_diffusion()
-    { return "ConvectionDiffusion"+_suffix; }
+    { return "ConvectionDiffusion"+_suffix(); }
 
   private:
 
     static std::string physics_name_delimiter()
     { return ":"; }
 
-    static std::string _suffix;
+    static std::string& _suffix()
```

Comment:
If I change the function name to `suffix`, then it clashes with the parameter`suffix` on line 39 (`set_suffix()`). Any preferences for what to rename that?